tests/test_field.py

```diff
@@ -30,7 +30,15 @@ def test_field_from_netcdf_variables():
         f3 = Field.from_netcdf(filename, variable, dims)
 
 
-@pytest.mark.parametrize("with_timestamps", [True, False])
+@pytest.mark.parametrize(
+    "with_timestamps",
+    [
+        pytest.param(
+            True, marks=[pytest.mark.v4remove, pytest.mark.xfail(reason="GH1918. This test should be rewritten.")]
+        ),
+        False,
+    ],
+)
```

Comment:
I would just remove the test altogether; it's not relevant anymore

Reply:
I was a bit thrown by the name `test_field_from_netcdf` so I thought the intention here was more general than just the timestamps. I don't mind either way - should I still remove?

Reply:
Ah I see. Yes, let’s leave the test in for now then. We could remove the (failing) `with_timestamps` argument to keep the code somewhat cleaner, as we don’t expect timestamps to return later?